config/standard/tasks.js

```diff
@@ -23,7 +28,7 @@
         days: 0,
         start: 0,
         end: 6,
-        dueDate: function(r, event) {
+        dueDate: function(event, c, r) {
```

Comment:
This comment is more of a place holder. I know we chatted about getting the rest of the medic-conf issues to 3.5 project. If PR [157](https://github.com/medic/medic-conf/pull/157/) isn't included this needs to be changed.